scripts/Job.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: haic-telemetry-export
+spec:
+  template:
+    metadata:
+      name: haic-telemetry-export
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: haic-telemetry-export-runtime
+        image: gcr.io/vorvan/h2oai/haic-telemetry-exporter:latest
```

Comment:
@ChathurindaRanasinghe QQ: where is the Dockefile for the container?

Reply:
Here @mmalohlava https://github.com/h2oai/cloud-utils/pull/35/files#diff-313bf94e58b04ca85b029fe151ffec6bd19167331cec7dbf66af017ded5e9654

Reply:
thank you! pls document here in this repo as well if possible.